kong/plugins/ai-prompt-template/handler.lua

```diff
@@ -0,0 +1,131 @@
+local _M = {}
+
+-- imports
+local kong_meta      = require "kong.meta"
+local templater      = require("kong.plugins.ai-prompt-template.templater"):new()
+local fmt            = string.format
+local parse_url      = require("socket.url").parse
+local byte           = string.byte
+local sub            = string.sub
+local cjson          = require("cjson.safe")
+--
+
+_M.PRIORITY = 773
+_M.VERSION = kong_meta.version
+
+
+local log_entry_keys = {
+  REQUEST_BODY = "ai.payload.original_request",
+}
+
+local function bad_request(msg)
+  kong.log.warn(msg)
+  return kong.response.exit(400, { error = { message = msg } })
```

Comment:
Instead of hardcoding it we can use the `ngx.HTTP_BAD_REQUEST` constant from https://github.com/openresty/lua-nginx-module?tab=readme-ov-file#http-status-constants